src/app/client/src/app/modules/core/components/prominent-filter/prominent-filter.component.ts

```diff
@@ -228,6 +229,13 @@ export class ProminentFilterComponent implements OnInit, OnDestroy {
   isObject(val) { return typeof val === 'object'; }
 
   applyFilters() {
+    const compareFormInputQueryparams = _.isEqual(this.formInputData,  this.queryParams);
+    if (compareFormInputQueryparams) {
+      this.isFiltered = true;
```

Comment:
cant we make the isFiltered = false as default and if compareFormInputQueryparams make it true

Reply:
yes, its false when declared